eng/Build.props

```diff
@@ -166,6 +166,7 @@
                         Exclude="
                           @(ProjectToBuild);
                           @(ProjectToExclude);
+                          $(RepoRoot)src\Razor\test\testassets\**\*.*proj;
```

Comment:
These projects will still be restored by Microsoft.NET.Sdk.Razor but we will skip building them since they will be built during tests.